Index: stage2/rootfs/etc/sysctl.conf
===================================================================
--- stage2.orig/rootfs/etc/sysctl.conf
+++ stage2/rootfs/etc/sysctl.conf
@@ -25,12 +25,12 @@
 #net.ipv4.tcp_syncookies=1
 
 # Uncomment the next line to enable packet forwarding for IPv4
-#net.ipv4.ip_forward=1
+net.ipv4.ip_forward=1
 
 # Uncomment the next line to enable packet forwarding for IPv6
 #  Enabling this option disables Stateless Address Autoconfiguration
 #  based on Router Advertisements for this host
-#net.ipv6.conf.all.forwarding=1
+net.ipv6.conf.all.forwarding=1
 
 
 ###################################################################
