Index: stage2/rootfs/etc/systemd/system/dnscrypt-proxy.socket
===================================================================
--- stage2.orig/rootfs/etc/systemd/system/dnscrypt-proxy.socket
+++ stage2/rootfs/etc/systemd/system/dnscrypt-proxy.socket
@@ -8,6 +8,8 @@ Wants=dnscrypt-proxy-resolvconf.service
 [Socket]
 ListenStream=127.0.2.1:53
 ListenDatagram=127.0.2.1:53
+ListenStream=[::1]:53
+ListenDatagram=[::1]:53
 NoDelay=true
 DeferAcceptSec=1
 
